src/main/java/org/openrewrite/java/template/processor/RefasterTemplateProcessor.java

```diff
@@ -622,12 +623,12 @@ private Set<String> getImportsAsStrings(Map<TemplateDescriptor, Set<String>> imp
                     return null;
                 }
 
-                return new Precondition.Or(
-                        preconditions.values().stream()
-                                .map(it -> new Precondition.And(it, indent + 4))
-                                .collect(toSet())
-                        , indent + 4)
-                        .prune();
+                return
+                        new Precondition.Or(
```

Comment:
Why the newline between these keywords?
```suggestion
                return new Precondition.Or(
```